README.md

```diff
@@ -56,27 +56,99 @@ Code quality:
     ```
 
 1. Start app with required services
-    ```shell script
+    ```shell
     make runserver
     ```
 
 1. Service will be available at `http://127.0.0.1:8042/`
 
-## System requires
+## Start the project for development without Docker
+1. Clone the repo and go to the project directory
+    ```shell
+    git clone git@github.com:Uma-Tech/parrot.git && \
```

Comment:
Nit: There's no real need for `&&` in this example.

Reply:
```suggestion
    git clone git@github.com:Uma-Tech/parrot.git
```